table/updates.go

```diff
@@ -25,15 +25,26 @@ import (
 )
 
 const (
-	updateSpec             = "add-spec"
-	updateAddSchema        = "add-schema"
-	updateSnapshot         = "add-snapshot"
-	updateSortOrder        = "add-sort-order"
-	updateAssignUUID       = "assign-uuid"
-	updateDefaultSpec      = "set-default-spec"
-	updateCurrentSchema    = "set-current-schema"
-	updateDefaultSortOrder = "set-default-sort-order"
-	updateUpgradeFormat    = "upgrade-format-version"
+	// These are the various update actions defined in the iceberg spec
```

Comment:
Shift the docstring to be above the `const` so that it gets picked up as the docstring for the whole block of constants, not just for that one.